### PR TITLE
Use JWT for Auth to ingest API

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose

Fixes https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/secondary-analysis/481

---

### Changes
Create a JWT from the CaaS service account and send that in requests to ingest to replace using Auth0.

---
### Review Instructions
- No instructions.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [x] This PR updated docstrings or documentation.
- [x] This PR applied code style guidelines:
    - [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) Python code.
    - Mint WDL style guide for WDL.
- [x] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
- Note the JWT creation code is the same as the DCP Auth Client in Lira -- we should consider consolidating them at a future point.

### CODE REVIEW

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

File: adapter_pipelines/submit.wdl [hunk 1]
```diff
@@ -64,6 +64,7 @@ task create_submission {
   Boolean record_http
   String pipeline_tools_version
   Boolean add_md5s
+  String runtime_environment
```
Comment (rexwangcc) [inline adapter_pipelines/submit.wdl]: Reminder: we might need to change the submit_stub as well so the test can pass

Comment (samanehsan) [reply adapter_pipelines/submit.wdl]: This PR does not add any new parameters to the submit subworkflow, so we do not need to change the submit_stub wdl

Comment (rexwangcc) [reply adapter_pipelines/submit.wdl]: Oh, i must be looking at the wrong variable name!

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
The tests are passing! ✅I'll change the version of the pipeline tools docker image before merging the PR.

**Pull request merged**